spark/src/main/scala/geotrellis/spark/filter/ToSpatial.scala

```diff
@@ -34,17 +36,16 @@ object ToSpatial {
     * to define additional constraints on Metadata. M should
     * depend on a K type (M[K]), and two type classes should
     * be provided: [[geotrellis.util.Component]], to extract key bounds
-    * from M[K], and [[geotrellis.util.Functor]] to map
-    * M[K] to M[SpatialKey].
+    * from M[K], and cats.Functor to map M[K] to M[SpatialKey].
     *
     * For those reading the source code directly,
-    * {{{K: λ[α => M[α] => Functor[M, α]]: λ[α => Component[M[α], Bounds[α]]}}}
+    * {{{K: λ[α => M[α] => Functor[M]]: λ[α => Component[M[α], Bounds[α]]}}}
```

Comment:
Functor is already passed simpler, so this line can be replaced by `{{{K: λ[α => Component[M[α], Bounds[α]]}}}`.
And below you can remove `ev1: M[K] => Functor[M]`. That intention here was to show what is `λ` here in types.